README.md

```diff
@@ -1,12 +1,94 @@
+  [![PRs closed](https://img.shields.io/github/issues-pr-closed/CrewMates-Open-Source/EduHelp)](https://github.com/CrewMates-Open-Source/EduHelp/pulls)
+  ![Repo size](https://img.shields.io/github/repo-size/CrewMates-Open-Source/EduHelp)
+  [![Open Source Love svg2](https://badges.frapsoft.com/os/v2/open-source.svg?v=103)](https://github.com/ellerbrock/open-source-badges/)
+ 
+[![forthebadge](https://forthebadge.com/images/badges/built-by-developers.svg)](https://forthebadge.com)
+[![forthebadge](https://forthebadge.com/images/badges/built-with-love.svg)](https://forthebadge.com)
+
 ## What It Does ? :thinking:
-**EduHelp** simply mean to make it easier for you to *record any online session*, be it an online lecture, meetings, youtube tutorials, etc. But we are not simply making a web extension or desktop app that records your screen or gives you an audio file in the end. We mean to *save the slide show* or screen presentation going on during those meets as well as the *text file of the entire conversation* that took place. This Project will not only help normal people but also *differently abled people* because the audio is converted to text and a file containing its summary is saved in .txt file as well as *braille scripted file*. The braille file can be given as an input to refreshable braille display (rbd) machines. The audio files can also be used by a blind person for taking note of what happened during class. The screenshots are stored in a .docx file and can be used by a deaf person for revisiting concepts they might miss during classes.
+**EduHelp** simply mean to make it easier for you to *record any online session*, be it an online lecture, meetings, youtube tutorials, etc. But we are not simply making a web extension or desktop app that records your screen or gives you an audio file in the end. We mean to *save the slide show* or screen presentation going on during those meets as well as the *text file of the entire conversation* that took place. This Project will not only help normal people but also *disabled persons* because the audio is converted to text and a file containing its summary is saved in .txt file as well as *braille scripted file*.The braille file can be given as an input to refreshable braille display (rbd) machines. The audio files can also be used by a blind person for taking note of what happened during class. The screenshots are stored in a .docx file and can be used by a deaf person for revisiting concepts they might miss during classes.
 <h2 align="center"> AMAZING!!😮:star_struck:RIGHT?</h2>
 
+## Demo
+Want to know more about EduHelp. Just click the Image Below :point_down:
+<p align="center">
+  <a href="https://youtu.be/qgMU8F1QX5g"><img src="https://img.youtube.com/vi/qgMU8F1QX5g/maxresdefault.jpg" width=500 height=200></img></a>
+</p>
+
+## What Do We Think Of Implementation ?
```

Comment:
Don't capitalize the 1st letter of every word. Do it for only the 1st word. Rest all words should be in lower case.